### PR TITLE
Added Custom Themes Support And Settings Modal

### PR DESCRIPTION
**Pull request by techwithanirudh**

### CODE REVIEW

File: client/public/assets/favicon-32x32.png
Comment (danny-avila) [inline client/public/assets/favicon-32x32.png:1]: Is this necessary?

File: api/server/routes/endpoints.js
Comment (danny-avila) [inline api/server/routes/endpoints.js:1]: Many files are changed with small formatting issues. I believe you’re not adhering to the eslint config and maybe using your own linter rules

File: .replit
Comment (danny-avila) [inline .replit:1]: Is this necessary? Should this be gitignored?

File: client/src/style.css
Comment (danny-avila) [inline client/src/style.css:1]: There is no way I can approve this PR with so many changes to the main style sheet and not enough support as to why

**Review by danny-avila — CHANGES_REQUESTED** [review]

File: client/public/assets/favicon-32x32.png
Comment (techwithanirudh) [reply client/public/assets/favicon-32x32.png:1]: No...

File: api/server/routes/endpoints.js
Comment (techwithanirudh) [reply api/server/routes/endpoints.js:1]: Sorry, I will relint it

**Comment by danny-avila** [comment]
I appreciate your recent efforts in contributing to our project with this PR. The time and dedication you've put into it is valued. However, after careful consideration, it became clear that the scope and depth of this Pull Request warrant revisiting our approach to new features.

Given the scope-creep we're noticing in the project, we are adopting a deeper strategy/process for proposing new features. This is driven by our aim to prevent:

    Rejecting PRs, causing contributors to feel frustrated and undervalued.
    Managing chaos arising from conflicting priorities that disrupt ongoing tasks in various areas of the project.
    Accepting code that does not align with our architectural and design direction, leading to refactoring and maintenance burdens.

As a part of this new process, we're introducing a proposal outline for feature requests. This should cover aspects such as the description of changes, the intention behind them, the necessity of the feature, affected areas of the application, designs if relevant to UI, and so on. We believe this will streamline our collaborative efforts by fostering discussions and iterations on-feature proposals before any actual work begins.

In light of our new standards and goals, we are unable to accept your current PR. We noticed that there were 45 files changed with many personal linting rules ignoring the eslint config in our repo. Moreover, the main stylesheet had 1500 lines removed, and there were .replit code and personal shell scripts included that are not typically utilized. Additionally, please note that we are transitioning our client code to TypeScript. Therefore, any incoming JavaScript code would increase the workload on our team.

We encourage you to consider the new proposal outline process and starting a discussion in the github discussions section after seeking alignment with our strategic direction. This will help ensure that we create high-quality and coherent outputs that bring value to our project and maintain long-term project health. 

Right now, we are more likely to approve PRs for actions on our trello, which we will transition to the GitHub projects section, and PRs smaller in scope.

We truly appreciate your understanding as we work together to build a strong project foundation moving forward. Please do not hesitate to reach out if you have any questions or need further clarification on the new process.

Kind regards,

File: client/src/style.css
Comment (techwithanirudh) [reply client/src/style.css:1]: All of the css in the main stylesheet were not required as tailwind will handle all of the css. Some values were hard coded in the css which broke themes support. As such the majority of the css was removed

File: api/server/routes/endpoints.js
Comment (techwithanirudh) [reply api/server/routes/endpoints.js:1]: Relinted it

File: client/public/assets/favicon-32x32.png
Comment (techwithanirudh) [reply client/public/assets/favicon-32x32.png:1]: Resolved in latest commit

File: api/server/routes/endpoints.js
Comment (techwithanirudh) [reply api/server/routes/endpoints.js:1]: Resolved in latest commit

File: .replit
Comment (techwithanirudh) [reply .replit:1]: Resolved in latest commit

File: client/src/style.css
Comment (techwithanirudh) [reply client/src/style.css:1]: Resolved in latest commit

**Pull request closed without merging**